test/web3_util.py

```diff
@@ -0,0 +1,34 @@
+"""Util functions for invoking and calling Web3 contracts"""
+
+from test.test_endpoints import load_file_content
+
+import json
+from web3 import Web3
+
+
+def web3_invoke(function, url, contract_address, abi_path, *inputs):
```

Comment:
Did you name this function this `invoke` because of experience with Starknet? It might be better to use `transact` because Ethereum doesn't use the call/invoke distinction. Also, is there a difference between using `transact` and `send`?